code/modules/overmap/internet/internet_uplink.dm

```diff
@@ -161,6 +162,7 @@ var/global/list/internet_uplinks = list()
 	light_color = COLOR_GREEN
 	idle_power_usage = 250
 	active_power_usage = 500
+	base_type = /obj/machinery/computer/internet_uplink
```

Comment:
Does this actually work? Won't the circuit not match the construct state if you do this? I'm kind of surprised it passes unit tests (most likely because it's not present on any map).

Reply:
I doubt it breaks anything, because that ran through unit tests downstream on a map we had it on.
Base type is just the buildable variant of this machine to fall back to when disassembling it.
It's essentially unneeded here though. I probably forgot to remove it after I got rid of the buildable type for this one.

Reply:
> Unbuildable component variants in `additional_spawn_components` are questionable: they mean that mapped objects will spawn with them, but they will not be dropped when deconstructed, so you have to source your own component when rebuilding (a _built_ object will not spawn them). Better to either place them in `uncreated_component_parts` (will spawn in built objects too) or place the `/buildable` variant in `additional_spawn_components`.

Honestly, it's a bit weird to get reviewed over code that isn't even in this PR and that was already reviewed?

But yeah I know. That's something that I've fixed downstream for ages. But apparently didn't make it here. I was mostly just pushing the missing recipes here.

Reply:
It works but it's redundant, because that's not a buildable subtype. So I'll remove that.